assets/src/components/site-scan-results/site-scan-results.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+import { ExternalLink, VisuallyHidden } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { Selectable } from '../selectable';
+
+/**
+ * Renders a panel with a site scan results.
+ *
+ * @param {Object}  props                   Component props.
+ * @param {Object}  props.children          Component children.
+ * @param {number}  props.count             Incompatibilities count.
+ * @param {string}  props.className         Additional class names.
+ * @param {Element} props.icon              Panel icon.
+ * @param {string}  props.title             Panel title.
+ * @param {string}  props.validatedUrlsLink URL to the Validated URLs page.
+ */
+export function SiteScanResults( {
+	children,
+	count,
+	className,
+	icon,
+	title,
+	validatedUrlsLink,
+} ) {
+	return (
+		<Selectable className={ classnames( 'site-scan-results', className ) }>
+			<div className="site-scan-results__header">
+				{ icon }
+				<p
+					className="site-scan-results__heading"
+					data-badge-content={ count }
+				>
+					{ title }
+					<VisuallyHidden as="span">
+						{ `(${ count })` }
+					</VisuallyHidden>
+				</p>
+			</div>
+			<div className="site-scan-results__content">
+				{ children }
+				{ validatedUrlsLink && (
+					<p className="site-scan-results__cta">
+						<ExternalLink href={ validatedUrlsLink }>
+							{ __( 'AMP Validated URLs page', 'amp' ) }
+						</ExternalLink>
```

Comment:
```suggestion
						<ExternalLink href={ validatedUrlsLink }>
							{ __( 'Review Validated URLs', 'amp' ) }
						</ExternalLink>
```

Also, I'm not sure `ExternalLink` is entirely right. I guess we're using it also in the AMP Validation sidebar in the block editor. But it's not an external link. It could be treated the same as the “Reopen Wizard” link in the intro.

Reply:
Yes, I think this persisted as an `ExternalLink` from the Onboarding Wizard. There it makes sense to open a new tab instead of getting a user out of the Wizard flow, without completing it first. On the Settings screen it doesn't make sense to open a new tab. I'll fix it.

Reply:
Addressed in d5a3103